packages/node/src/api-spec.ts

```diff
@@ -0,0 +1,21 @@
+import { Node } from "@counterfactual/types";
```

Comment:
The name of this file made me think it's a test file at first glance lol

Reply:
Maybe api router is a better name?

Reply:
👍

Reply:
I thought the name was gonna be changed before merging? @snario